composer.json

```diff
@@ -212,7 +212,7 @@
         },
         "ps_apiresources": {
             "type": "vcs",
-            "url": "https://github.com/jolelievre/ps_apiresources.git"
+            "url": "https://github.com/PrestaShop/ps_apiresources.git"
```

Comment:
Hmm we can remove the block here? Because we will use the default repository.

Reply:
No because we're still gonna have many branches to integrate We will remove it when we release the first version of the module